src/database/seed.ts

```diff
@@ -0,0 +1,93 @@
+import { getExistingOrCreateNewUser } from '../application/service';
```

Comment:
Not sure about the name of this file.  It seems to add test data, but it sound like it will seed production?

Reply:
Yeah.. Since we delete the database on every run in production for now I just want to create some data that allows use to have different states to test.

As tests/testTypes etc keep coming we can add them here to have a state in production we can play with. 

Before goLive we should kill this file

Reply:
Maybe just rename it `test-seed.ts` or something?